core/downloadfromdcthread.cpp

```diff
@@ -49,21 +50,38 @@ void DownloadThread::run()
 	qDebug() << "Finishing the thread" << errorText << "dives downloaded" << downloadTable.nr;
 }
 
+static void fill_supported_mobile_list()
+{
```

Comment:
actually, the comment below @ line 58 should go right here... just call this 
fill_mobile_device_supported_list()
or something - it's Android and iOS that are triggering the restriction